lib/Mojo/Base.pm

```diff
@@ -61,6 +61,10 @@ sub attr {
   }
 }
 
+sub can_roles {
+  return !!(eval 'require Role::Tiny' && $Role::Tiny::VERSION >= 2.000005);
```

Comment:
This should be a constant and the `can_roles` method is just for exposure outside of the class. C.f. https://github.com/kraih/mojo/blob/be50b20ae6c80df78a2594fdd198c19364f649ad/lib/Mojo/IOLoop/Client.pm#L13-L15